README.rst

```diff
@@ -220,3 +220,8 @@ DALI was built with major contributions from Trevor Gale, Przemek Tredak, Simon
 
 .. |License| image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
    :target: https://opensource.org/licenses/Apache-2.0
+
+.. |Documentation| image:: https://img.shields.io/badge/nvidia%20dali-documentation-brightgreen.svg?longCache=true
+   :target: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/
```

Comment:
Maybe we should point to https://docs.nvidia.com/deeplearning/sdk/dali-master-branch-user-guide/docs/, as it is master branch?
@ptrendx ?

Reply:
BTW - it should be `Nvidia DALI` I believe.

Reply:
![image](https://user-images.githubusercontent.com/10923599/48422868-306f0f80-e768-11e8-954e-0bc8b6c1e728.png)

@JanuszL done